### PR TITLE
[PGEC-45] Rule 14: Bundle already offered

### PR DESCRIPTION
**Pull request by jacopocarlini**
#### List of Changes

added check when PSP offers a bundle: return conflict if the bundle has already been offered

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/afm/marketplacebe/service/BundleOfferService.java [hunk 1]
```diff
@@ -85,6 +75,13 @@ public List<BundleOffered> sendBundleOffer(String idPsp, String idBundle, CiFisc
 
         List<BundleOffered> bundleOfferedList = new ArrayList<>();
         ciFiscalCodeList.getCiFiscalCodeList().forEach(fiscalCode -> {
+            // check if the bundle has already been offered
+            var ciBundle = ciBundleRepository.findByIdBundleAndCiFiscalCode(idBundle, fiscalCode);
+            if (ciBundle.isPresent()) {
```
Comment (aomegax) [inline src/main/java/it/pagopa/afm/marketplacebe/service/BundleOfferService.java:80]: could a bundle offer be send to all or nobody? 
how do you think it?

reading the code the behaviour seems be ambiguous

**Review by aomegax — COMMENTED** [review]
only a little check to do

File: src/main/java/it/pagopa/afm/marketplacebe/service/BundleOfferService.java (hunk 1 above)
Comment (jacopocarlini) [reply src/main/java/it/pagopa/afm/marketplacebe/service/BundleOfferService.java]: fixed

**Review by aomegax — APPROVED** [review]
👍

**Pull request merged**